packages/renderer/src/lib/images/SettingsIcon.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+export let size = '24';
```

Comment:
Nit: if size is being set in NavPage anyway, 1em is probably a better default size for reuse.